health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -227,65 +171,57 @@ export const UICustomizations = {
-
       //adding tenantId to moduleSearchCriteria
       data.body.inbox.moduleSearchCriteria.tenantId = Digit.ULBService.getCurrentTenantId();
-
-      //setting limit and offset becoz somehow they are not getting set in muster inbox
-      data.body.inbox.limit = data.state.tableForm.limit;
-      data.body.inbox.offset = data.state.tableForm.offset;
-      delete data.state;
+      //setting limit and offset becoz somehow they are not getting set in muster inbox 
+      data.body.inbox .limit = data.state.tableForm.limit
+      data.body.inbox.offset = data.state.tableForm.offset
+      delete data.state
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)